memsimplekv/keyvalue.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package memsimplekv
+
+import (
+	"sync"
+	"time"
+
+	"golang.org/x/net/context"
+	errgo "gopkg.in/errgo.v1"
+
+	"github.com/juju/simplekv"
+)
+
+// NewStores returns a new Stores instance.
```

Comment:
This comment seems to contradict the comment below. Also I see little point in exposing this level of indirection here.
Why not just have
```
func NewStore() simplekv.Store {
	return &kvstore{
		data: make(map[string][]byte),
	}
}
```
and make everything work around that level of interface.

Reply:
done